Patches the gazelle-generated BUILD.bazel file so that
we can export all Go files in the package for further analysis
and code generation.

--- a/pkg/golinters/BUILD.bazel
+++ b/pkg/golinters/BUILD.bazel
@@ -1,5 +1,11 @@
 load("@io_bazel_rules_go//go:def.bzl", "go_library")
 
+filegroup(
+    name = "all_go_files",
+    srcs = glob(["*.go"]),
+    visibility = ["//visibility:public"],
+)
+
 go_library(
     name = "golinters",
     srcs = ["typecheck.go"],
